manylinux_v2_ubuntu/Dockerfile

```diff
@@ -0,0 +1,81 @@
+ARG BASE_IMAGE=nvidia/cuda:11.8.0-devel-ubuntu20.04
+FROM ${BASE_IMAGE}
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+ENV PYENV_ROOT="/pyenv" \
+    PATH="/pyenv/bin:/pyenv/shims:$PATH"
+
+RUN apt update -y && apt install -y jq build-essential software-properties-common wget gcc zlib1g-dev libbz2-dev libssl-dev libreadline-dev libsqlite3-dev libffi-dev curl git libncurses5-dev libnuma-dev openssh-client libcudnn8-dev zip libopenblas-dev liblapack-dev protobuf-compiler autoconf automake libtool cmake && rm -rf /var/lib/apt/lists/*
+
+RUN add-apt-repository ppa:git-core/ppa && add-apt-repository ppa:ubuntu-toolchain-r/test && apt update -y && apt install -y git gcc-9 g++-9 && add-apt-repository -r ppa:git-core/ppa && add-apt-repository -r ppa:ubuntu-toolchain-r/test
```

Comment:
Should we use GCC 11?

Reply:
We can't unfortunately, that will start using newer libstdc++ symbols.